interfaces/builtin/ofono.go

```diff
@@ -325,7 +335,12 @@ func (iface *ofonoInterface) DBusPermanentSlot(spec *dbus.Specification, plug *i
 }
 
 func (iface *ofonoInterface) UDevPermanentSlot(spec *udev.Specification, slot *interfaces.Slot) error {
-	spec.AddSnippet(ofonoPermanentSlotUDev)
+	udevRule := ofonoPermanentSlotUDev
+	for appName := range slot.Apps {
+		tag := udevSnapSecurityName(slot.Snap.Name(), appName)
+		udevRule += fmt.Sprintf(ofonoPermanentSlotUDevTag, tag)
```

Comment:
I'd love the same replacement logic that we use elsewhere simply because this will lessen the friction to unify everything at some point in the future.